libr/include/r_anal.h

```diff
@@ -611,6 +598,7 @@ typedef struct r_anal_hint_cb_t {
 typedef struct r_anal_t {
 	RArchConfig *config;
 	int lineswidth; // asm.lines.width
+	bool pseudo;    // asm.pseudo
```

Comment:
Couldnt find a way to pass RAsm.pseudo into an RAnalPlugin. Will remove this if it's possible otherwise.

Reply:
this change will break the ABI, so it's not suposed to be merged on a patch release.

imho asm->pseudo makes "sense", because rparse will be merged into rasm, when all the asm plugins get moved to anal. 

So the non-breaking abi solution to this is to use RAnal.coreb.cfgGetI("asm.pseudo")

Reply:
but ideally the logic of "pseudo" should be separated from the disassembler logic that lives in r_anal now